Packages/vcs/vcs/VCS_validation_functions.py

```diff
@@ -243,6 +243,18 @@ def checkListOfNumbers(self, name, value, minvalue=None,
     return list(value)
 
 
+def checkValidOption(self, name, value, options):
```

Comment:
I think you could have sed checkInStringList, it would have made it more consistent with the rest of API where you can pass string or number.
see:
https://github.com/UV-CDAT/uvcdat/blob/b69d09d59a2d07853c5f4f2fe6987610d141b07a/Packages/vcs/vcs/VCS_validation_functions.py#L1321